downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/GOAGeneratorUtilities.java

```diff
@@ -207,7 +224,7 @@ public static void outputGOAFile() throws IOException {
      * @throws IOException -- If file or targetDirectory do not exist, this will be thrown.
      */
     public static void moveFile(String targetDirectory) throws IOException {
-        targetDirectory += GOAGeneratorConstants.GOA_FILENAME;
-        Files.move(Paths.get(GOAGeneratorConstants.GOA_FILENAME), Paths.get(targetDirectory), StandardCopyOption.REPLACE_EXISTING);
+        String updatedTargetDirectory = targetDirectory + GOAGeneratorConstants.GOA_FILENAME + ".gz";
```

Comment:
Could this be called 'targetFilePath' since it is the directory plus the file name?